File: zendrive-ui/src/api/models/stats/StatsRequest.ts
import { StorageConfig } from "@apiModels/metafile/StorageConfig";

export class StatsRequest {
	path: string;
	storageConfig: StorageConfig;

	constructor(path: string, storageConfig: StorageConfig) {
		this.path = path;
		this.storageConfig = storageConfig;
	}
}

File: zendrive-api/src/main/java/com/zendrive/api/rest/models/dto/stats/StatsRequest.java
package com.zendrive.api.rest.models.dto.stats;

import com.zendrive.api.core.model.task.StorageConfig;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class StatsRequest {
	private String path;
	private StorageConfig storageConfig;
}
